index.js

```diff
@@ -1423,6 +1423,11 @@ export const createJavaBom = async (path, options) => {
         }
       } // for
       if (pkgList.length) {
+        if (parentComponent.components && parentComponent.components.length) {
+          for (const subProj of parentComponent.components) {
+            pkgList = pkgList.filter(pkg => !(pkg.group === subProj.group && pkg.name === subProj.name && pkg.version === subProj.version))
```

Comment:
Could we make use of `bom-ref` here?

Reply:
Would probably be easier, but I think it wasn't set on the items in one of the lists... But let me check to make sure.